### PR TITLE
OP-464 use isEmpty()

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-464

There are some small spelling changes in this PR too; as I was reviewing the files I noticed them and couldn't help myself; I had to change them.

### CODE REVIEW

File: src/main/java/org/isf/medstockmovtype/rest/MedStockMovementTypeController.java [hunk 1]
```diff
@@ -70,13 +70,13 @@
 	public ResponseEntity<List<MovementTypeDTO>> getMedicaldsrstockmovType() throws OHServiceException {
 		LOGGER.info("Retrieving all the movement types ...");
 		List<MovementType> movementTypes = manager.getMedicaldsrstockmovType();
-		List<MovementTypeDTO> mappedMvments = mapper.map2DTOList(movementTypes);
-		if(mappedMvments.isEmpty()) {
+		List<MovementTypeDTO> mappedMovments = mapper.map2DTOList(movementTypes);
+		if (mappedMovments.isEmpty()) {
```
Comment (mwithi) [inline src/main/java/org/isf/medstockmovtype/rest/MedStockMovementTypeController.java:74]: is the 'e' of movements missing intentionally?

**Review by mwithi — APPROVED** [review]
just one comment, thanks!

File: src/main/java/org/isf/medstockmovtype/rest/MedStockMovementTypeController.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/medstockmovtype/rest/MedStockMovementTypeController.java:74]: Nope; I'll fix it.

Comment (dbmalkovsky) [reply src/main/java/org/isf/medstockmovtype/rest/MedStockMovementTypeController.java]: Done.

**Pull request merged**